web/src/app/components/modal/modal.component.html

```diff
@@ -141,10 +143,10 @@ <h4 ibmModalHeaderHeading> <svg ibmIconFolderAdd size="32"></svg> Add a new case
         <button class="bx--btn bx--btn--secondary" (click)="closeModal() ">
             Cancel
         </button>
-   <button  *ngIf="stepCounter === 'First'" [disabled]="!this.defendantAndCaseForm.valid || this.validateDefendantRace.value[0] == '' || this.validateDefendantGender.value[0] == ''"  class="bx--btn bx--btn--primary" modal-primary-focus (click)="goNextOrSave() ">
+   <button *ngIf="stepCounter === 'First'" [disabled]="!this.defendantAndCaseForm.valid || this.validateDefendantRace.value[0] == '' || this.validateDefendantGender.value[0] == ''"  class="bx--btn bx--btn--primary" modal-primary-focus (click)="goNextOrSave() ">
             {{textForButton}}
         </button>
-        <button  *ngIf="stepCounter === 'Second'" [disabled]="this.validateFilter.value === null"  class="bx--btn bx--btn--primary" modal-primary-focus (click)="closeModal()">
+        <button  *ngIf="stepCounter === 'Second'" [disabled]="this.validateFilter.value === null"  class="bx--btn bx--btn--primary" modal-primary-focus (click)="goNextOrSave()">
```

Comment:
Changed the button to save instead of just closing the modal so I could make sure the data was getting through.